modules/google-analytics.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Module Name: Google Analytics
+ * Module Description: MODULE DESCRIPTION COPY.
+ * First Introduced: 4.4
+ * Sort Order: 37
+ * Requires Connection: No
+ * Auto Activate: No
+ * Feature: Engagement
+ * Additional Search Queries: webmaster, google, analytics, console
+ */
+
+/**
+ * Removes the Google Analytics plugin settings page
+ */
+function removeSettingsPage() {
```

Comment:
Could you use only lowercase names for function names, as per [the WP coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions)?